tests/end_to_end/test_suites/tr_with_fedeval_tests.py

```diff
@@ -99,9 +100,9 @@ def test_eval_federation_via_dockerized_workspace(request, fx_federation_tr_dws)
         num_rounds=1,
     ), "Federation completion failed"
 
-    new_metric_file_path = os.path.join(new_fed_obj.aggregator.workspace_path, "logs", "aggregator_metrics.txt")
-    model_accuracy_eval = get_aggregated_accuracy(new_metric_file_path)
-    log.info(f"Model accuracy during evaluation only on prev trained model is : {model_accuracy_eval})")
+    tensor_db_file_path = os.path.join(new_fed_obj.aggregator.workspace_path, "local_state", "tensor.db")
+    best_model_accuracy_eval = get_best_accuracy(tensor_db_file_path)
+    log.info(f"Model accuracy post {request.config.num_rounds} rounds: {best_model_accuracy}")
 
     # verify that the model accuracy is similar to the previous model accuracy max of 1% difference
-    assert abs(model_accuracy - model_accuracy_eval) <= 0.01, "Model accuracy is not similar to the previous model accuracy"
+    assert abs(best_model_accuracy - best_model_accuracy_eval) <= 0.01, "Model accuracy is not similar to the previous model accuracy"
```

Comment:
1% is significant in data science. If logically the expectation is to get same value under floating point error, this tolerance should be around 1e-5.